app/views/browse/show.html.erb

```diff
@@ -20,7 +20,7 @@
   } %>
 <% end %>
 
-<%= render "shared/browse_header", { margin_bottom: page.slug == "benefits" ? 7 : 9 } do %>
+<%= render "shared/browse_header", { margin_bottom: [ "benefits", "business" ].include?(page.slug) ? 7 : 9 } do %>
```

Comment:
As an aside, I feel a bit uncomfortable applying this kind of logic to solve a small layout problem (margin size). I wonder if this could be addressed in the design system so that different margin sizes aren't necessary?

Reply:
> Now that we have more than one slug, could we create a reusable variable for the slugs? This would make the intent of the logic more obvious and could be [reused on line 34](https://github.com/alphagov/collections/pull/3437/commits/e4c7d506ea5fe75619a15f4bba3f3639e8d537ec#diff-f4daef91a07e97d1c8caf0cd07d782bdbcd5ea86ea94e757b52d21244e78fec7R34). E.g `slugs_with_action_link = ...`. Another approach may be to add a new helper method with the `include` logic too which DRYs this is out further.
> 
> Also [surprised the linter didn't complain](https://www.rubydoc.info/github/bbatsov/RuboCop/RuboCop/Cop/Style/WordArray), maybe because there are spaces between the brackets... but the array would ideally be formatted like: `%w[benefits business]` for an array of strings, with more than one value. Thanks!

@1pretz1 - thanks for the feedback. I've moved the code into a helper and fixed the array formatting issues (currently the changes are under a separate commit but I'll merge into one if things look okay)

Reply:
Nice thanks @jon-kirwan! My last suggestion would be to rename the generic slugs method to something more explicit,  maybe `SLUGS` -> `ACITON_LINK_SLUGS` and `is_matching_slug?` -> `display_action_links_for_slug?`, as we're only using this conditional for action links. What do you think?

Reply:
@1pretz1 thanks, yep good suggestions. I've included both!